### DISCUSSION TITLE
[@types/node-zendesk] PaginablePayload next_page and previous_page types should be string not number

### DISCUSSION THREAD

**Discussion by stormsa** [DefinitelyTyped/DefinitelyTyped#59723] [Issues with a @types package]
Hello, 

According to the Zendesk docs here https://developer.zendesk.com/rest_api/docs/support/introduction#pagination, the next_page and previous_page fields are url's (strings) not numbers, but in the PaginablePayload interface they have type number | null.

`
// index.d.ts line 1091
export interface PaginablePayload {
    next_page: number | null;
    previous_page: number | null;
    count: number;
}
`
should be

`
export interface PaginablePayload {
    next_page: string | null;
    previous_page: string | null;
    count: number;
}
`

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
I just opened #59735
